src/seq_mv/csr_spgemm_device_rowest.c

```diff
@@ -57,12 +61,18 @@ void rownnz_naive_rowi(HYPRE_Int rowi, HYPRE_Int lane_id, HYPRE_Int *ia, HYPRE_I
 
 template <char type, HYPRE_Int NUM_WARPS_PER_BLOCK>
 __global__
-void csr_spmm_rownnz_naive(HYPRE_Int M, /*HYPRE_Int K,*/ HYPRE_Int N, HYPRE_Int *ia, HYPRE_Int *ja,
-                           HYPRE_Int *ib, HYPRE_Int *jb, HYPRE_Int *rcL, HYPRE_Int *rcU)
+void csr_spmm_rownnz_naive(HYPRE_Int  M,
+                           HYPRE_Int  N,
+                           HYPRE_Int *ia,
+                           HYPRE_Int *ja,
+                           HYPRE_Int *ib,
+                           HYPRE_Int *jb,
+                           HYPRE_Int *rcL,
+                           HYPRE_Int *rcU)
 {
    const HYPRE_Int num_warps = NUM_WARPS_PER_BLOCK * gridDim.x;
    /* warp id inside the block */
-   const HYPRE_Int warp_id = get_warp_id();
+   const HYPRE_Int warp_id = get_group_id();
```

Comment:
since `blockDim.x * blockDim.y == HYPRE_WARP_SIZE`, `get_group_id == get_warp_id`, which is more efficient.